### PR TITLE
Update application dependencies

### PR DESCRIPTION
**Pull request by jgarber623**
~~**Note:** Opening this draft PR against the `fix-apps-and-spec-errors` branch (PR #65). Will update and/or create a new PR against `master` when that PR is merged.~~

This pull request updates the application's dependencies in a semi-conservative fashion. All non-pending specs pass and RuboCop reports only a single offense (in `spec/views/submissions/show.html.erb_spec.rb`) that may be addressed later.

Closes #59
Closes #60
Closes #61
Closes #62
Closes #63
Closes #64

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-require: rubocop-rspec
+require: 
+  - rubocop-rails
+  - rubocop-rspec
```
Comment (jgarber623) [inline .rubocop.yml:3]: Rails cops were removed from RuboCop in [version 0.72](https://github.com/rubocop-hq/rubocop/releases/tag/v0.72.0).

File: .rubocop.yml [hunk 2]
```diff
@@ -15,17 +17,14 @@ AllCops:
     - '.git/**/*'
 
 IndentationConsistency:
-  EnforcedStyle: 'rails'
+  EnforcedStyle: 'indented_internal_methods'
```
Comment (jgarber623) [inline .rubocop.yml:20]: Another change in [RuboCop version 0.72](https://github.com/rubocop-hq/rubocop/releases/tag/v0.72.0).

File: spec/factories/submissions.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-    url 'MyText'
-    html 'MyText'
-    base_url 'MyText'
-    save_html false
-    render_html_in_page false
+    url { 'MyText' }
+    html { 'MyText' }
+    base_url { 'MyText' }
+    save_html { false }
+    render_html_in_page { false }
```
Comment (jgarber623) [inline spec/factories/submissions.rb:7]: Newer versions of RuboCop use this syntax.

**Comment by veganstraightedge** [comment]
@jgarber623 I no longer maintain these repos. Could ask @aaronpk et al to review your PRs and/or give you commit bit, please? I would hate to have you waiting around on me and me not showing up for you.

**Comment by jgarber623** [comment]
@veganstraightedge No worries, bud! Thanks for the head's up. I've tagged @aaronpk for a review.

**Comment by jgarber623** [comment]
It's been four years, so… closing.

**Pull request closed without merging**